lib/random_word.rb

```diff
@@ -71,6 +79,10 @@ def exclude_list
       @exclude_list ||= []
     end
 
+    def rng=(r)
+      word_list.rng = r
```

Comment:
I don't think this is going to work as intended. That `word_list` ivar is a temporary holding place that gets overwritten every time a new random word list is created. 

We could do this in the `#noun` and `#adjs` methods the way we do constraints (https://github.com/openlogic/random-word/blob/master/lib/random_word.rb#L77 and https://github.com/openlogic/random-word/blob/master/lib/random_word.rb#L84 respectively). Or refactor `EachRandomly` in to a decorator that wraps the word list. :)